.vscode/deno_resolve_npm_imports.json

```diff
@@ -1,11 +1,13 @@
 {
-  "// Resolve NPM imports for `packages/remix-deno`.": "",
-  "// This import map is used solely for the denoland.vscode-deno extension.": "",
-  "// Remix does not support import maps.": "",
-  "// Dependency management is done through `npm` and `node_modules/` instead.": "",
-  "// Deno-only dependencies may be imported via URL imports (without using import maps).": "",
+  "comment": [
+    "Resolve NPM imports for `packages/remix-deno`.",
+    "This import map is used solely for the denoland.vscode-deno extension.",
+    "Remix does not support import maps.",
+    "Dependency management is done through `npm` and `node_modules/` instead.",
+    "Deno-only dependencies may be imported via URL imports (without using import maps)."
+  ],
```

Comment:
Moved it here because otherwise the typechecker gives lots of scary-looking warnings.